dist/gcc-compatible/configure

```diff
@@ -79,23 +88,23 @@ int main() {
   return 0;
 }
 EOF
-  $CC -I. -I../kremlin/include -I../kremlin/kremlib/dist/minimal -march=native -O3 $file -o $out && $out > $out.test
-  $CC -I. -I../kremlin/include -I../kremlin/kremlib/dist/minimal -DBROKEN_INTRINSICS -march=native -O3 $file -o $out && $out > $out.ref
+  $CC $CROSS_CFLAGS -I. -I../kremlin/include -I../kremlin/kremlib/dist/minimal -march=native -O3 $file -o $out && $out > $out.test
```

Comment:
does this -march=native override the CROSS_FLAGS? same below and in a bunch of other invocations

Reply:
I guess you have to rely on either $CROSS_CFLAGS to specify it and, otherwise, to be -march=native ... if this is the default behavior of gcc (i.e. ignore subsequent -march flags) then I guess we're good